README.md

```diff
@@ -1,36 +1,35 @@
 ![logo](https://raw.githubusercontent.com/greenelab/BioBombe/master/logo.png)
 
-# Biological Interpretation of Serially Compressed Gene Expression Features using Network Projection
+# Sequential Compression of Gene Expression Data Across Latent Space Dimensions
 
 **Gregory Way and Casey Greene 2018**
 
 **University of Pennsylvania**
 
 [![DOI](https://zenodo.org/badge/126377943.svg)](https://zenodo.org/badge/latestdoi/126377943)
 
-The repository stores data and data processing modules to enable compressed gene expression feature interpretation.
+The repository stores data and data processing modules to sequentially compress gene expression data.
 
-Named after the [mechanical device developed by cryptologists in World War II](https://en.wikipedia.org/wiki/Bombe) to decipher secret messages sent by [Enigma machines](https://en.wikipedia.org/wiki/Enigma_machine), BioBombe is used to decipher hidden signals in gene expression data.
-Inspired by the number-crunching knobs of [Alan Turing's](https://en.wikipedia.org/wiki/Alan_Turing) device, BioBombe serially compresses gene expression input with increasing bottleneck dimensionality and deciphers the learned compressed features using biological network projection approaches.
+Named after the [mechanical device developed by cryptologists in World War II](https://en.wikipedia.org/wiki/Bombe) to decipher secret messages sent by [Enigma machines](https://en.wikipedia.org/wiki/Enigma_machine), BioBombe is used to enhance biological signatures in gene expression data.
+Inspired by the number-crunching knobs of [Alan Turing's](https://en.wikipedia.org/wiki/Alan_Turing) device, BioBombe sequentially compresses gene expression input across latent dimensionalities and deciphers the the biological signals embedded within compressed gene expression features.
```

Comment:
Maybe a bit nit picky, not sure what you mean by enhance.  I thought that biobome was used more to explore the signals that were found from different model architectures to help with interpretability and point to a window of latent space features that are most useful.

Reply:
> more to explore the signals that were found from different model architectures to help with interpretability and **point to a window of latent space features that are most useful.** (:point_left: YES!)

That is what I mean by enhance! The reader will have to engage in the same way when we make this point.